source/Outfit.cpp

```diff
@@ -286,6 +288,9 @@ void Outfit::Load(const DataNode &node)
 			child.PrintTrace("Skipping unrecognized attribute:");
 	}
 
+	if(pluralName.empty())
+		pluralName = displayName + 's';
+
```

Comment:
Can we not also the displayName here like this instead of up above?
The only reason I see to default the displayName first instead of checking if it is empty at this stage and then defaulting it to the unique name of the outfit would be to support doing something like `"display name" ""` in the outfit definition to set an empty display name. But, if that is done, and `"plural name"` was not explicitly set, or it was set to be similarly empty, it would get updated at this point to `s`, which seems a bit silly. If a person really wants no display name, they probably want the plural name to be the same: nothing.
Do we even want to allow empty displayNames?

Reply:
Look at the commit I just added, better like that?

Reply:
One could just explicitly provide an empty plural name as well.